src/features/filters/server/filterToPrisma.ts

```diff
@@ -30,13 +34,19 @@ export function filterToPrismaSql(
 
     const colPrisma = Prisma.raw(col.internal);
     const operatorPrisma =
-      filter.operator in operatorReplacements
+      filter.type === "arrayOptions"
         ? Prisma.raw(
-            operatorReplacements[
-              filter.operator as keyof typeof operatorReplacements
+            arrayOperatorReplacements[
```

Comment:
operatorPrisma wont be used if all of or none of.
- For all of: cant we use @> as operator?
- For none of: cant we use && as operator and then add another variable to the final return string like "func"? If func is not empty, we wrap the final prisma statement in `func(...)`